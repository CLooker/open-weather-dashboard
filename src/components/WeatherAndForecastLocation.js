import React, { PureComponent } from 'react';
import PropTypes from 'prop-types';

export default class WeatherAndForecastLocation extends PureComponent {
  static propTypes = {
    name: PropTypes.string
  };

  render() {
    const { name } = this.props;
    return <h1>Weather and Forecast in {name}</h1>;
  }
}
