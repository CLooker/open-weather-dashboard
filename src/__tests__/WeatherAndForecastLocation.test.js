import React from 'react';
import { shallow } from 'enzyme';
import WeatherAndForecastLocation from '../components/WeatherAndForecastLocation';

describe('Loading', () => {
  it('should render correctly', () => {
    const props = {
      name: 'Chicago'
    };
    const component = shallow(<WeatherAndForecastLocation {...props} />);
    expect(component).toMatchSnapshot();
  });
});
