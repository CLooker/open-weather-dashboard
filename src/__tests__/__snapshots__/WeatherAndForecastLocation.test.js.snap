// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should render correctly 1`] = `
"<h1>
  Weather and Forecast in 
  Chicago
</h1>"
`;
